enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk_setup/components/SelectEmbedResourceStep.tsx

```diff
@@ -23,23 +26,31 @@ export const SelectEmbedResourceStep = () => {
     updateSettings,
     recentDashboards,
     recentQuestions,
+    recentCollections,
     addRecentItem,
   } = useSdkIframeEmbedSetupContext();
 
   const [isPickerOpen, { open: openPicker, close: closePicker }] =
     useDisclosure(false);
 
-  // Only dashboard and charts allow selecting resources.
-  if (experience !== "dashboard" && experience !== "chart") {
+  // Exploration does not allow selecting resources.
+  if (experience === "exploration") {
```

Comment:
collection, dashboard and chart all allow selecting resources, so excluding exploration makes more sense.